data/json/vehicleparts/vp_flags.json

```diff
@@ -587,5 +587,10 @@
   {
     "id": "WINDSHIELD",
     "type": "json_flag"
+  },
+  {
+    "id": "LIMITED_SPACE",
```

Comment:
This isn't a great name, something like, "normal_human_size" or something is more accurate.
Also there needs to be another one for "oversized" to prevent regular sized humans from using fixtures scaled for huge mutants.

Reply:
Will do, I will split this into two flags and change the name.

Reply:
3 flags:
CRAMPED_FOR_MUTANTS implements previous cramping behavior for huge muties.
MUTANT_OK means a mutant can start up these controls. Applied to boat paddles, cart handle, wheelchair rims, XL controls.
HUMAN_UNUSABLE means if you aren't huge you can't start the vehicle. XL controls only.